src/workloads/docs/CrudActorFSM.yml

```diff
@@ -68,23 +70,24 @@ Actors:
       # Duration. These keywords behave the same as in Phase Control, but operate per state (i.e.,
       # per actor instance as it goes through the state machine.
       Repeat: 1
-      Operations:
+
       # A state can have (optional) a set of operations that execute upon entering the state. This
       # state updates a specific document to indicate it is in state On.
 
       # Note: Following the analogy to phases -- phases may be blocking or non-blocking. States are
-      # always non-blocking. When the phase ends, the state machine will finish any open operations,
-      # and then exit. It if is waiting for a delay, it will exit immediately. This is required as
-      # there may be states with long delays.
-      # Question: Should we support having a single fire operation with repeating operations? Here I
-      # could invent a case where the following operation is repeat 1, but there is another
-      # operation that repeats once a second while in this state. I don't know how to represent
-      # that.
+      # always non-blocking, in the sense that any sleepBefore, SleepAfter delays will be
+      # interrupted to close the phase. However, any active operations will be finished.
+
```

Comment:
@rtimmons does this added text make sense to your question? Both above and below this comment.